python_client/pyrefinebio/http.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+def get(endpoint, params=None):
+    url = ""
+
+    if endpoint.startswith("http"):
+        url = endpoint
+    else:
+        url = "https://api.refine.bio/v1/" + endpoint + "/"
+
+    try:
+        response = requests.get(url, params=params)
+
+        response.raise_for_status()
+        results = response.json()
+        # cache results?
+
+    except requests.exceptions.HTTPError:
+        logging.error("GET %s failed", endpoint)
+        return None
+
+    return results
+
+
+def post(endpoint, payload=None):
+    try:
+        response = requests.post(
+            "http://localhost:8000/v1/" + endpoint + "/",
```

Comment:
We should probably make this a constant in one place, and then for testing you can switch it back and forth as necessary.

Reply:
Can you load from an environment variable if available? Makes testing even easier. And I agree with a constant in one place for the default :)